utils/utils_windows.go

```diff
@@ -28,7 +28,10 @@ func SyncFS(syncPath string) error {
 	if err != nil {
 		return err
 	}
-	defer syscall.CloseHandle(handle)
+	defer func() {
+		//nolint:errcheck,gosec // CloseHandle errors can be ignored in defer
+		syscall.CloseHandle(handle)
```

Comment:
```suggestion
		goutils.UncheckedError(syscall.CloseHandle(handle))
```
As with similar in above files, Optional!